var/spack/repos/builtin/packages/cosma/package.py

```diff
@@ -25,7 +25,7 @@ class Cosma(CMakePackage):
     version("2.6.6", sha256="1604be101e77192fbcc5551236bc87888d336e402f5409bbdd9dea900401cc37")
     version("2.6.5", sha256="10d9b7ecc1ce44ec5b9e0c0bf89278a63029912ec3ea99661be8576b553ececf")
     version("2.6.4", sha256="6d7bd5e3005874af9542a329c93e7ccd29ca1a5573dae27618fac2704fa2b6ab")
-    version("2.6.3", sha256="8ca96ca41458f1e9d0da70d524c5a03c677dba7238d23a578f852163b6d45ac9")
+    version("2.6.3", sha256="c2a3735ea8f860930bea6706d968497d72a1be0498c689b5bc4a951ffc2d1146")
```

Comment:
Any idea why the sha256 changed?

Reply:
Or has it been incorrect since its introduction?

Reply:
Thanks for checking @tldahlgren . The new sha256 is correct. I'm almost sure it was incorrect since introduction, but nobody noticed because we shortly after release bugfix releases.